CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # jsonld ChangeLog
 
+## 6.0.0 -
+
+### Changed
+- **BREAKING**: No longer manually define browser global `JsonLdProcessor`;
+  this is left to Webpack and other bundlers. (Currently, this is messing up
+  downstream bundling and karma tests.)
```

Comment:
The suggestion below is to use `globalThis`. But if not, I don't understand what "left to Webpack and other bundlers" means.  How would that work?  Also a better explanation than "messing up downstream" would be preferred.  It's not clear what that means here or in commits.